tests/test_geodesic.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import plenoptic as po
+import pytest
+import torch
+from torchvision.transforms.functional import center_crop
+
+from conftest import DATA_DIR, DEVICE
+
+imageA = po.load_images(DATA_DIR + '/256/reptil_skin.pgm')
+imgA = center_crop(imageA, [64]).to(DEVICE)
```

Comment:
Does this have to be reptile skin? If you use one of the image fixtures, we save a bit of time in loading images in. (reptile skin could also be made a fixture)